client/css/style.css

```diff
@@ -762,15 +762,15 @@ button {
 }
 
 #chat .messages {
-	display: table;
-	table-layout: fixed;
-	width: 100%;
 	padding: 10px 0;
 }
 
 #chat .msg {
 	word-wrap: break-word;
 	word-break: break-word; /* Webkit-specific */
+	display: flex;
```

Comment:
with [autoprefixer](https://github.com/postcss/autoprefixer) and #609 this could be done magically behind the scenes :)